src/main/json/r6OperatorCounters.json

```diff
@@ -1503,6 +1503,33 @@
                     "properties": {
                         "Description": "Dokkaebi can use her Logic Bomb to ring Caveria's phone. This will reveal Caveria's position when roaming even if Silent Step is active."
                     }
+                },
+                {
+                  "id": "143",
+                  "type": "Hard Counter",
+                  "startNode": "48".
+                  "endNode": "47",
+                  "properties": {
+                      "Description": "Nøkk's HEL will remove her image from Mozzie's drones."
+                  }
+                },
+                {
+                    "id": "144",
+                    "type": "Hard Counter",
+                    "startNode": "39",
+                    "endNode": "47",
+                    "properties": {
+                        "Description": "Thatcher's EMP Grenades can destroy Mozzie's Pests"
```

Comment:
Could you add a full stop/period to the end of this sentence, please? Do you know if Thatcher destroys/disables hacked drones? Might be good to mention if he does/doesn't. Not sure if the answer :)